profiling-ffi/src/profiles.rs

```diff
@@ -368,19 +369,47 @@ pub unsafe extern "C" fn ddog_Profile_set_endpoint<'a>(
     profile.add_endpoint(local_root_span_id, endpoint);
 }
 
+/// Count the number of times an endpoint has been seen.
+///
+/// # Arguments
+/// * `profile` - a reference to the profile that will contain the samples.
+/// * `endpoint` - the value of the endpoint label to add for matching samples.
+///
+/// # Safety
+/// The `profile` ptr must point to a valid Profile object created by this
+/// module.
+/// This call is _NOT_ thread-safe.
+#[no_mangle]
+pub unsafe extern "C" fn ddog_Profile_add_endpoint_count(
```

Comment:
Minor: Do you mind if we call this `ddog_prof_Profile_add_endpoint_count`? I'm working on a PR to namespace-prefix the profiling APIs and this would be one less that would need the rename :)

See also [libdatadog API naming review doc](https://docs.google.com/spreadsheets/d/1iuuSCJ0NcSOpo2RVvkQ8f8ugz2XlEbbpFZLVOg6WioE/edit#gid=0) (internal link only, sorry!)

Reply:
no pb ;) will do

Reply:
✔️